README.md

```diff
@@ -21,23 +21,25 @@ example demonstrates the usage of HTTPLimiter for rate-limiting access
 to an http.Handler to 20 requests per path per minute with bursts of
 up to 5 additional requests:
 
-	store, err := memstore.New(65536)
-	if err != nil {
-		log.Fatal(err)
-	}
+```go
+store, err := memstore.New(65536)
+if err != nil {
+	log.Fatal(err)
```

Comment:
Oh weird, GitHub was rendering the text incorrectly on my screen.

<img width="337" alt="screenshot 2016-11-12 08 17 03" src="https://cloud.githubusercontent.com/assets/96890/20239222/7826cbae-a8b0-11e6-8d53-bb6253b1459e.png">

Just moved computers and it looks okay though. Thanks!